src/main/java/pl/company/foodatu/meals/domain/MealFactory.java

```diff
@@ -0,0 +1,15 @@
+package pl.company.foodatu.meals.domain;
+
+import java.util.List;
+import java.util.UUID;
+
+class MealFactory {
+
+    Meal createMeal(String name, List<Product> products) {
+        return new Meal(
+                UUID.randomUUID().toString(),
```

Comment:
Just for viz: having hardcoded `random` in the code can sometimes be a problem. There are cases (but not here), when you want to have control over what values are generated in tests. In that case we could have `IdGenerator` interface and two implementations of it (random and controlled)